sdk/src/oauth/state-machines/debug-oauth-2025-11-25.ts

```
@@ -1852,20 +1855,19 @@ export const createDebugOAuthStateMachine = (
             }
 
             // Build the token request body as an object (will be shown in HTTP history)
+            const previewClientAuth = buildTokenRequestClientAuth({
+              clientId: state.clientId,
+              clientSecret: state.clientSecret,
+              tokenEndpointAuthMethod: state.tokenEndpointAuthMethod,
+            });
```

Biome (2.5.1)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)